packages/openid4vc/src/openid4vc-issuer/repository/OpenId4VcIssuanceSessionRecord.ts

```diff
@@ -134,6 +147,16 @@ export interface OpenId4VcIssuanceSessionRecordProps {
    */
   presentation?: OpenId4VcIssuanceSessionPresentation
 
+  /**
+   * When the issuance is deferred, this is the transaction id that is used to
+   * link the deferred credential requests to the issuance session.
+   */
+  // TODO: I'm assuming it is not possible to have multiple deferred credential
+  // requests within the same issuance session. If that is not the case, this needs
+  // to be reworked.
```

Comment:
i think it's possible theoretically? If you offer both drivers license and PID and both need a few days to process? If we want to allow it is another question, I'm not sure 🤔